### ISSUE TITLE
USWDS - Bug: Reconsider Radio tile outline styling

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5487]
### Describe the bug

A request from a community member: 

I wanted to bring something to your attention and provide feedback on an accessibility issue. I was recently taking a quiz on a website that is following GSA's [digital.gov](http://digital.gov/) accessibility guidelines, but the design of the quiz made the content extremely difficult for me to read. Each quiz question answer had a black box around it which caused my eyes to lose focus.  My eyes could only focus on the space between the boxes and not on the text. This caused the bottom half of the text to look wavy and make it unreadable. When I used a piece of paper to cover the black lines, I could read the text without issue. Attached is an example:

![image](https://github.com/uswds/uswds/assets/93996430/bb772577-dd14-4fc7-b86c-436f98aa7fe5)

I did some research and discovered that this can be the experience of anyone who has a visual processing disorder (such as dyslexia). When I looked up the GSA guidelines, this design was used as a radio button component preview: https://designsystem.digital.gov/components/radio-buttons/#component-preview

I wanted to suggest that you reconsider using this type of design in the website guidelines as the black border makes the content unreadable for some.  A better idea would be to use a grey shadow box instead of a border.

I appreciate your consideration in updating the guidelines.  If you have any questions, please let me know!

### Steps to reproduce the bug

N/A

### Expected Behavior

USWDS components should not add unnecessary styles that hinder usability. 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mejiaj** [comment]
Changing from `confirmation` to `discussion` to figure out what to do next.

@amycole501 @alex-hull could you:

1. Provide further insight into the a11y requirements to this component?
2. Is there a certain contrast level for borders or similar elements for this?
3. Are there resources you can point us to related to visual processing (or any research really)?

**Comment by amyleadem** [comment]
Note: The tile border colors are defined with disabled color tokens. These settings received an update in 3.5.0, which made the border color darker. Note the color shifts between [3.4.0](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/release-3.4.0/?path=/story/components-form-inputs-radio--radio-tile) and [3.5.0](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/release-3.5.0/?path=/story/components-form-inputs-radio--radio-tile). This could play a role in the visual noise. 

![image](https://github.com/uswds/uswds/assets/93996430/353e03f1-6c4b-47ed-b72a-2c316364828b)

**Comment by amycole501** [comment]
To address James' questions:

1. Provide further insight into the a11y requirements to this component?
Radio button audit has been completed; https://github.com/uswds/uswds/issues/5504

2. Is there a certain contrast level for borders or similar elements for this?
Yes: SC 1.4.11  https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

Given that the "tile" option of the radio button isn't critical for accessibility, we should add guidance that indicates when to use or not use the "tile" option.  I recommend we conduct some usability testing on both options to see which is more user friendly.

3. Are there resources you can point us to related to visual processing (or any research really)?
This is a great question and I have found a myriad of resources I'd like to dig further into. I will make a ticket for myself to do this research and report back findings.

**Comment by brunerae** [comment]
@thisisdano this is ready to be closed!
